# Japanese translation for linuxmint
# Copyright (c) 2013 Rosetta Contributors and Canonical Ltd 2013
# This file is distributed under the same license as the linuxmint package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2013.
#
msgid ""
msgstr ""
"Project-Id-Version: linuxmint\n"
"Report-Msgid-Bugs-To: https://launchpad.net/~linuxmint-translation-team-"
"japanese\n"
"POT-Creation-Date: 2015-11-26 15:10+0000\n"
"PO-Revision-Date: 2017-02-28 13:31+0000\n"
"Last-Translator: Hiroshi Tagawa <Unknown>\n"
"Language-Team: Japanese <https://launchpad.net/~linuxmint-translation-team-"
"japanese>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Launchpad-Export-Date: 2017-06-28 07:49+0000\n"
"X-Generator: Launchpad (build 18416)\n"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:34 generate_desktop_files:26
#: generate_desktop_files:42
msgid "Driver Manager"
msgstr "ドライバマネージャ"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:38
msgid "Re_vert"
msgstr "元に戻す (_V)"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:40
msgid "_Apply Changes"
msgstr "変更の適用 (_A)"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:42
msgid "_Cancel"
msgstr "キャンセル (_C)"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:44
msgid "_Restart..."
msgstr "再起動 (_R)..."

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:69
msgid ""
"Drivers cannot be installed.\n"
"Please connect to the Internet or insert the Linux Mint installation DVD (or "
"USB stick)."
msgstr ""

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:188
msgid "Applying changes..."
msgstr "変更を適用しています..."

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:334
msgid "This device is using the recommended driver."
msgstr "このデバイスは推奨されたドライバを利用しています。"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:335
msgid "This device is using an alternative driver."
msgstr "このデバイスは代替のドライバを利用しています。"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:336
msgid "This device is using a manually-installed driver."
msgstr "このデバイスは手動インストールされたドライバを利用しています。"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:337
msgid "This device is not working."
msgstr "このデバイスは動作していません。"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:346
msgid "Continue using a manually installed driver"
msgstr "手動インストールされたドライバーを使い続けます"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:373
msgid "Version"
msgstr "バージョン"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:376
#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:381
msgid "recommended"
msgstr "推奨"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:378
msgid "open-source"
msgstr "オープンソース"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:384
msgid "requires a connection to the Internet"
msgstr "インターネットへの接続が必要です"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:408
msgid "Do not use the device"
msgstr "デバイスを使用しません"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:426
#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:427
#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:471
msgid "Unknown"
msgstr "不明"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:539
msgid "You need to restart the computer to complete the driver changes."
msgstr "ドライバの変更を完了するにはコンピュータを再起動する必要があります。"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:553
#, python-format
msgid "%(count)d proprietary driver in use."
msgid_plural "%(count)d proprietary drivers in use."
msgstr[0] "%(count)d 個のプロプライエタリドライバを利用しています。"

#: usr/lib/linuxmint/mintDrivers/mintDrivers.py:558
msgid "No proprietary drivers are in use."
msgstr "プロプライエタリドライバを利用していません。"

#: usr/bin/driver-manager:9
msgid "Please enter your password to launch the driver manager"
msgstr "ドライバマネージャを起動するにはパスワードを入力してください"

#: generate_desktop_files:26 generate_desktop_files:42
msgid "Manage the drivers for your devices"
msgstr "デバイスドライバを管理"
